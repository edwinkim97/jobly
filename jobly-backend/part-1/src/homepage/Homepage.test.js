import React from "react";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router";
import Home from "./Homepage";


it("matches snapshot", function () {
  const { asFragment } = render(
    <MemoryRouter>
      <Home />
    </MemoryRouter>,
  );
  expect(asFragment()).toMatchSnapshot();
});

