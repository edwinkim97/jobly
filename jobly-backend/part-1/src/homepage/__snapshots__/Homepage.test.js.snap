// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Homepage"
  >
    <div
      class="container text-center"
    >
      <h1
        class="mb-4 fw-bold"
      >
        Jobly
      </h1>
      <p
        class="lead"
      >
        All the jobs in one, convenient place.
      </p>
    </div>
  </div>
</DocumentFragment>
`;
